compiler/src/main/kotlin/dev/zacsweers/metro/compiler/fir/generators/ContributedInterfaceSupertypeGenerator.kt

```diff
@@ -67,31 +66,26 @@ internal class ContributedInterfaceSupertypeGenerator(session: FirSession) :
   private val inCompilationScopesToContributions:
     FirCache<FirSession, Map<ClassId, Set<ClassId>>, TypeResolveService> =
     session.firCachesFactory.createCache { session, typeResolver ->
-      val scopesToContributingClass = mutableMapOf<ClassId, MutableSet<ClassId>>()
```

Comment:
Sorry I kinda kludged some of this on main not realizing it would mess with this. But hopefully it's straightforward, my change was to basically simplify this cache to make it use scope `ClassId`s as the input and the value a `Set<ClassId>`, essentially promoting that `Map<ClassId, Set<ClassId>>` that was here before to be the signature of this FIR cache instead